src/renderer/hooks/useSingleTxFee.ts

```diff
@@ -0,0 +1,25 @@
+import { AssetAmount } from '@thorchain/asgardex-util'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import { useObservable, useObservableState } from 'observable-hooks'
+import { Observable } from 'rxjs'
+import * as Rx from 'rxjs/operators'
+
+import { TransferFees } from '../services/binance/types'
+
+const useSingleTxFee = (transferFees$: Observable<O.Option<TransferFees>>): O.Option<AssetAmount> =>
+  useObservableState(
+    useObservable(() =>
+      transferFees$.pipe(
+        Rx.map((fees) =>
+          FP.pipe(
+            fees,
+            O.map((f) => f.single)
+          )
+        )
+      )
+    ),
+    O.none
+  )
+
+export default useSingleTxFee
```

Comment:
what do you think about avoiding default exports ? 
it will allow IDE to suggest correct imports for autoimports and will force to import needed values explicitly

Reply:
Ok.